torch/csrc/jit/runtime/register_c10_ops.cpp

```diff
@@ -15,7 +15,6 @@ namespace {
 //      It should also handle autograd.
 Operator createOperatorFromC10(const c10::OperatorHandle& op) {
   return Operator(op, [op](Stack& stack) {
-    RECORD_FUNCTION(op.schema().name(), stack);
```

Comment:
Interesting, what's going on here?

Reply:
This is now covered by the dispatch wrapper when we do `callBoxed` below